lib/Rx/Operator/IsEmptyOperator.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rx\Operator;
+
+use Rx\Observer\CallbackObserver;
+use Rx\Operator\OperatorInterface;
+use Rx\ObservableInterface;
+use Rx\ObserverInterface;
+use Rx\Scheduler\ImmediateScheduler;
+use Rx\SchedulerInterface;
+
+class IsEmptyOperator implements OperatorInterface
+{
+    /**
+     * @param \Rx\ObservableInterface $observable
+     * @param \Rx\ObserverInterface $observer
+     * @param \Rx\SchedulerInterface $scheduler
+     * @return \Rx\DisposableInterface
+     */
+    public function __invoke(ObservableInterface $observable, ObserverInterface $observer, SchedulerInterface $scheduler = null)
+    {
+        return $observable->subscribeCallback(
```

Comment:
Should probably use `subscribe` here since `subscribeCallback` is not on the observable interface